### DISCUSSION TITLE
Any plans to support SK in PHP?

### DISCUSSION THREAD

**Discussion by rad8329** [microsoft/semantic-kernel#2966] [General]
I saw Microsoft is focussing in develop SK for Java, the SDK for .NET and Python are ready. Any plans for PHP?

I know Microsoft historically haven't been friendly with PHP, but it could be handled by us (community). Thanks in advance for your thoughts.

**Comment by evchaki** [comment]
@rad8329  - we do not have any plans to support PHP as a kernel.  However, you can deploy any of the kernels as a service and have PHP call over to one of the kernels.

**Comment by mayursaptal** [comment]
Hi all! 👋
I’ve ported Semantic Kernel to PHP — fully aligned with the original architecture and naming conventions.

🔗 Packagist: https://packagist.org/packages/mayursaptal/semantic-kernel-php
💻 GitHub: https://github.com/mayursaptal/semantic-kernel-php

Would love your thoughts and contributions from the community!
